### DISCUSSION TITLE
Week Days Win Rate

### DISCUSSION THREAD

**Discussion by Elitesparkle** [Heroes-Profile/heroesprofile#702] [Ideas]
I'd like to be able to see on which day of the week a given player is winning more games.

![image](https://github.com/Heroes-Profile/heroesprofile/assets/9367799/09cdc99b-9b38-4365-a7a6-f7f25d76d226)

**Comment by Zemill** [comment]
I added this, should show up on the site in about 5min.

I didn't like the tracking aspect of it, so it only shows up for your own account if you are logged in

**Comment by Elitesparkle** [comment]
Thank you! I have noticed that a lot of games are missing.

I have played way more Quick Match games across all Seasons.

<img width="665" height="648" alt="image" src="https://github.com/user-attachments/assets/2e9c3322-a38d-4206-97ec-c2b149aba7bb" />

I have played way more Storm League games across all Seasons.

<img width="665" height="648" alt="image" src="https://github.com/user-attachments/assets/d371b3d6-49cd-4366-afd1-2e4b6ac5fb97" />

For all other Game Modes the numbers seem to be correct.

**Reply by Zemill** [reply]
I think it messes up if your profile has already been cached.  I'll take a look tomorrow and then rebuild the bad data.  

Your unfiltered one looks correct right?  I tested your account when I was testing this

**Reply by Elitesparkle** [reply]
Yes, without filters the numbers in my profile look correct.

**Comment by Zemill** [comment]
I think this has been fixed

**Reply by Elitesparkle** [reply]
Perfect! I can see the correct numbers now.

**Comment by Elitesparkle** [comment]
By the way, due to the ads on the right, the alignment is a bit off.

<img width="881" height="1053" alt="image" src="https://github.com/user-attachments/assets/f4257a5e-089d-41eb-b234-ae3b546f365e" />

I think it looks better if both panels with Win Rates are left-aligned.

**Reply by Zemill** [reply]
This is being caused by the ad showing up, I could just stick another ad in there :)

**Reply by Zemill** [reply]
<img width="1128" height="1615" alt="image" src="https://github.com/user-attachments/assets/4427e1e3-98a2-48e7-9ad7-505ad3949eee" />
